### PR TITLE
INFRA-124 Add Jenkinsfile for New Jenkins - Payara4

### PR DESCRIPTION
**Pull request by Pandrex247**

### CODE REVIEW

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
New Jenkins test please

**Comment by Pandrex247** [comment]
This is now blocked on the Arquillian Connector not supporting JDK7

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+            }
+            post {
+                unstable {
+                    junit '**/target/surefire-reports/*.xml'
+                }
+            }
+        }
+        stage('Setup for EE7 Tests JDK7') {
+            tools {
+                jdk "zulu-8"
```
Comment (MarkWareham) [inline Jenkinsfile]: JDK 7?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+            }
+            steps {
+                setupDomain()
+            }
+        }
+        stage('Run EE7 Tests JDK7') {
```
Comment (MarkWareham) [inline Jenkinsfile]: You're testing JDK7 using EE7 samples built in JDK8. This may or may not be OK.

**Pull request merged**